### ISSUE TITLE
Add ability to configure text color for all messages

### ISSUE THREAD

**Issue by idevtech** [gitkraken/vscode-gitlens#3077]
For better accessibility it would be helpful to be able to set the color of messages. For example, Inline Blame is red text on a black background which is almost impossible to see for people w/ red/green color blindness (many males). Maybe this feature exists but I have not been able to find it.
